sdk/operationalinsights/arm-operationalinsights/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@azure/arm-operationalinsights",
   "author": "Microsoft Corporation",
   "description": "OperationalInsightsManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
I think you should bump the major version. there's breaking change in the dependency

Reply:
Not just the dependency, I believe the models & mappers are being changed too

Reply:
yeah, but anyway we should bump major version

Reply:
Yes :) Thats what I meant in https://github.com/Azure/azure-sdk-for-js/pull/9226#issuecomment-637284318